samples/bootloader/pm.yml

```diff
@@ -1,17 +1,17 @@
 #include <autoconf.h>
 
-b0_image:
+b0:
```

Comment:
Nothing wrong here, but cannot place comments in the commit message.

Please add a bit more description on what/why this commit does in the commit message, and secondly, I don't think this commit should refer the NCSDK-5567, as it is just a cleanup commit, from what I can tell.

Reply:
I added a description to the commit and updated the PR message. The partition naming was the cause of the bug where an invalid merged b0.hex/b0n.hex was created, so I believe it is relevant to NCSDK-5567. The last two commits were just for cleanup, though.

Reply:
> The partition naming was the cause of the bug where an invalid merged b0.hex/b0n.hex was created

thanks for clarifying.
With the updated commit message, this is much clearer.

> The last two commits were just for cleanup, though.

there is nothing wrong in adding cleanup commits to a PR.
It's highly appreciated to cleanup code as part of making other changes.

But to make it easier for reviewers to distinguish between commits that fixes issues and cleanup that are done because you are anyway making changes, then it's important to clearly identify the kind of commit in the commit message.